index.js

```diff
@@ -16,7 +17,8 @@ function startWatching({ watch, match, exec }, { verbose }) {
   return new Promise(resolve => {
     match = match || [ '**/*' ];
     const prefix = [ list('watch', watch), list('match', match), list('exec', exec) ].join(' ');
-    const gaze = new Gaze(watch);
+    console.log(JSON.stringify(options.gazeOptions));
```

Comment:
Could you please use `log()` function instead of `console.log()`? I would like to keep ability to have it completely silent.